woocommerce/google-listings-and-ads
# Remove PMax Migration Banners

### Changes proposed in this Pull Request:



PMax Migration banners were still showing even though the migration ended in September last year. 

This PR removes the PMax migration banner and it is aligned with the following PR https://github.com/woocommerce/google-listings-and-ads/pull/1798 that removed the inbox notifications about the migration. 

### Screenshots:

#### Dashboard
![image](https://user-images.githubusercontent.com/2488994/219106930-68043049-a242-4078-a496-7343a30a649e.png)

#### Reports
![image](https://user-images.githubusercontent.com/2488994/219107046-759f1d00-8f26-4f07-9422-fe5e53ddc0f8.png)

### Detailed test instructions:


1. Go to the dashboard page, and the banner should not be displayed.
2. Go to the reports page, and the banner should not be displayed.

If you would like to test that banners were showing in develop branch follow these steps:

 1. Remove the following items from the local storage (in case they were dismissed previously): 
-  `localStorage.removeItem('gla-is-before-migration-notice-dismissed')`
-  `localStorage.removeItem('gla-is-after-migration-notice-dismissed')`
- `localStorage.removeItem('gla-is-reports-migration-notice-dismissed')`
2. Go to the dashboard or the reports page and the banner should be displayed. 



### Additional details:


### Changelog entry

> Tweak - Remove unnecessary PMax migration banners
